.github/workflows/packages.yml

```
@@ -18,13 +18,13 @@ jobs:
     - name: Install dependencies
       run: sudo apt-get update; sudo apt-get -y install build-essential git ncurses-dev 
     - name: Run a one-line script
-      run: git clone -b v5.3.10 --depth 1 'https://gitlab.nic.cz/turris/turris-build' "$GITHUB_WORKSPACE"/../turris-build
+      run: git clone -b v5.4.0 --depth 1 'https://gitlab.nic.cz/turris/turris-build' "$GITHUB_WORKSPACE"/../turris-build
     - name: Update feeds.conf for local repo
       run: cd "$GITHUB_WORKSPACE"/../turris-build; sed -i "s|^src-git cesnet.*|src-git cesnet $GITHUB_WORKSPACE|" feeds.conf
     - name: Prepare build for Turris MOX
       run: cd "$GITHUB_WORKSPACE"/../turris-build; ./compile_pkgs prepare_tools -t mox
     - name: Compile ipfixprobe package
```

I'm thinking, if you don't want to change it like this to be more clear.
```suggestion
    - name: Compile luci-app-ipfixprobe package, which compiles ipfixprobe, too.
```